data/curated/IMO_data/intro.md

```diff
@@ -0,0 +1,10 @@
+
+The data for this week comes from International Mathematical Olympiad (IMO).
+
+> The International Mathematical Olympiad (IMO) is the World Championship Mathematics Competition for High School students and is held annually in a different country. The first IMO was held in 1959 in Romania, with 7 countries participating. It has gradually expanded to over 100 countries from 5 continents. The competition consists of 6 problems and is held over two consecutive days with 3 problems each.
+
+QUESTION?
+
```

Comment:
```suggestion
```
I'll update the template. The intention here is for you to *replace* this line with a question or three, but that definitely isn't obvious!